src/Evolution/DiscontinuousGalerkin/SlopeLimiters/Minmod.hpp

```diff
@@ -0,0 +1,309 @@
+///    It does not try to avoid limiting as much as `LambdaPiN`, but it allows
+///    larger slopes in the data than `Muscl`.
+///
+/// For all three types of minmod limiter the "total variation bound in the
+/// means" (TVBM) correction is implemented, enabling the limiter to avoid
+/// limiting away smooth extrema in the solution that would otherwise look like
+/// spurious oscillations. The limiter will not reduce the slope (but will still
+/// linearize) on elements where the slope is less than \f$m h^2\f$, where
+/// \f$m\f$ is the TVBM constant and \f$h\f$ is the size of the DG element.
+///
+/// The limiter acts in the `Frame::Logical` coordinates, because in these
+/// coordinates it is straightforward to formulate the algorithm. This means the
+/// limiter can operate on generic deformed grids. However, if the grid is too
+/// strongly deformed, some things can start to break down:
+/// 1. When an element is deformed so that the Jacobian (from `Frame::Logical`
+///    to `Frame::Inertial`) varies across the element, then the limiter fails
+///    to be conservative. In other words, the integral of a tensor `u` over the
+///    element will change after the limiter activates on `u`. This error is
+///    typically small.
+/// 2. When there is a sudden change in the size of the elements (perhaps at an
+///    h-refinement boundary, or at the boundary between two blocks with very
+///    different mappings), a smooth solution in `Frame::Inertial` can appear
+///    to have a kink in `Frame::Logical`. The Minmod implementation includes
+///    some (untested) tweaks that try to reduce spurious limiter activations
+///    near these fake kinks.
+///
+/// The limiter as implemented expects an element to have one neighbor in each
+/// direction, and therefore does not support h-refinement.
+///
+/// \tparam VolumeDim The number of spatial dimensions.
+/// \tparam Tags A typelist of tags specifying the tensors to limit.
+///
+/// \anchor cockburn_ref [1] B. Cockburn,
+/// Discontinuous Galerkin Methods for Convection-Dominated Problems,
+/// [Springer (1999)](https://doi.org/10.1007/978-3-662-03882-6_2)
+template <size_t VolumeDim, typename... Tags>
+class Minmod<VolumeDim, tmpl::list<Tags...>> {
+ public:
+  struct Type {
+    using type = MinmodType;
+    static constexpr OptionString help = {"Type of minmod"};
+  };
+  struct TvbmConstant {
+    using type = double;
+    static type default_value() { return 0.0; }
+    static type lower_bound() { return 0.0; }
+    static constexpr OptionString help = {"TVBM constant 'm'"};
+  };
+  using options = tmpl::list<Type, TvbmConstant>;
+  static constexpr OptionString help = {
+      "A minmod-based slope limiter.\n"
+      "The different types of minmod are more or less aggressive in trying to\n"
+      "reduce slopes. The TVBM correction allows the limiter to ignore "
+      "'small'\n"
+      "slopes, and helps to avoid limiting of smooth extrema in the "
+      "solution.\n"};
```

Comment:
Oops I have to fix the formatting here. @nilsdeppe is this OptionString more like what you hoped to see? I tried to give more details but what direction to take this.